containers/flatpak/test/test-browser

```diff
@@ -57,6 +58,8 @@ class BrowserTest(unittest.TestCase):
         self.assertIn("run-js", actions)
 
     def tearDown(self):
+        if shutil.which("import"):
+            subprocess.call(["import", "-window", "root", f"test-browser-{ self.id().split('.')[-1] }.png"])
```

Comment:
Is this going to break if we try to run the tests under Wayland?  I do that sometimes locally.

Reply:
So do I :grin:  It's certainly got going to succeed, but as it's (deliberately) not `check_call()` it will not fail the test.

Reply:
It seems to work under wayland